google_fonts/lib/src/google_fonts_base.dart

```diff
@@ -104,23 +103,25 @@ Future<ByteData> readLocalFont(String name) async {
 ///   "700italic": "http://fonts.gstatic.com/s/alegreyasc/v11/taiRGmRtCJ62-O0HhNEa-Z6q4Sk0SK-UEGKDBz4.ttf",
 /// }
 String fontUrlKey(TextStyle textStyle) {
-  if (textStyle == null) {
+  if (textStyle == null ||
+      (textStyle.fontWeight == FontWeight.w400 &&
```

Comment:
can you make a var for isRegular to make this a bit more readable?

Reply:
Good call, Done.